wagtail/core/tests/test_audit_log.py

```diff
@@ -1,280 +1,265 @@
-from datetime import datetime, timedelta
+from datetime import timedelta
```

Comment:
It appears that you've moved wagtail/admin/tests/test_audit_log.py here, and overwritten the existing wagtail/core/tests/test_audit_log.py.